source/renderer/app/containers/wallet/dialogs/paper-wallet-certificate/InstructionsDialogContainer.js

```diff
@@ -0,0 +1,40 @@
+// @flow
+import React, { Component } from 'react';
+import { remote } from 'electron';
+import { observer, inject } from 'mobx-react';
+import InstructionsDialog from '../../../../components/wallet/paper-wallet-certificate/InstructionsDialog';
+import type { InjectedDialogContainerProps } from '../../../../types/injectedPropsType';
+
+type Props = InjectedDialogContainerProps;
+
+@inject('stores', 'actions') @observer
+export default class InstructionsDialogContainer extends Component<Props> {
+  static defaultProps = { actions: null, stores: null, children: null, onClose: () => {} };
+
+  onPrint = () => {
+    const filePath = remote.dialog.showSaveDialog({
+      defaultPath: '~/paper-wallet-certificate.pdf',
```

Comment:
We need to see if this works on Windows.

Reply:
@DominikGuzei @darko-mijic @tomothespian this is now fixed!
This value can be default title so no need to reference folder structure: https://electronjs.org/docs/api/dialog#dialogshowsavedialogbrowserwindow-options-callback

Reply:
...therefore I just changed this line to:
```
defaultPath: 'paper-wallet-certificate.pdf',
```